docs/MapColonies/Raster/services/PolygonParts/README.md

```diff
@@ -285,12 +322,17 @@ For convenience we'll add outputFormat parameter as `application/json` to each o
 
 </details>
 
-The response is the XSD (in xml response) describing the `orthophoto_mosaic_base_orthophoto_best_polygon_parts` `FeatureType`.
+
 
 This is a description of a orthophoto_mosaic_base_orthophoto_best_polygon_parts feature that include all polygon parts of the `ORTHOPHOTO_MOSAIC_BASE-OrthophotoBest` catalog layer, it contains these properties, each property has its own type, nillable flag value and min\max occurs - these describing the requireness of the property and possible repeatness.
 
+
+
 Now that we hold the structure of the `orthophoto_mosaic_base_orthophoto_best_polygon_parts` FeatureType we're able to query layer's polygon parts features by a set of parameters using the `GetFeature` operation, let's see some examples:
 
+
+### GetFeature GET basic request with count restricts
```

Comment:
```suggestion
#### Pagination
```

Reply:
And add another example for full use of pagination.

Reply:
done